client/orderbook/orderbook.go

```diff
@@ -366,21 +361,15 @@ func (ob *OrderBook) updateRemaining(note *msgjson.UpdateRemainingNote, cached b
 
 	ob.ordersMtx.Lock()
 	ord, found := ob.orders[oid]
-	var newOrder *Order
-	if found {
-		newOrder = ord.copy()
-		newOrder.Quantity = note.Remaining
-		ob.orders[oid] = newOrder
-	}
```

Comment:
Updating the Order instance in the bookSide that contains it takes care of this.  `bookSide` is now in charge of `Quantity` access exclusively.